src/routes/api.js

```diff
@@ -56,5 +57,8 @@ export default class APIRoutes {
     const paymentApi = api.basePath("/payments");
     paymentApi.post("/manual-payments", Payment.ManualPaymentsController.create);
     paymentApi.patch("/manual-payments/:id", Payment.ManualPaymentsController.update);
+
+    const salesayaNamespaceApi = api.basePath("/salesaya");
+    salesayaNamespaceApi.get("/products/search", Salesaya.ProductSearchController.index);
```

Comment:
https://topdev.vn/blog/restful-api-la-gi/
```suggestion
    salesayaNamespaceApi.get("/product_searches", Salesaya.ProductSearchController.index);
```